tests/integration_tests/reports/api_tests.py

```diff
@@ -52,17 +53,67 @@
 
 class TestReportSchedulesApi(SupersetTestCase):
     @pytest.fixture()
-    def create_working_report_schedule(self):
+    def create_working_admin_report_schedule(self):
         with self.create_app().app_context():
 
             admin_user = self.get_user("admin")
+            chart = db.session.query(Slice).first()
+            example_db = get_example_database()
+
+            report_schedule = insert_report_schedule(
+                type=ReportScheduleType.ALERT,
+                name="name_admin_working",
+                crontab="* * * * *",
+                sql="SELECT value from table",
+                description="Report working",
+                chart=chart,
+                database=example_db,
+                owners=[admin_user],
+                last_state=ReportState.WORKING,
+            )
+
+            yield
+
+            db.session.delete(report_schedule)
+            db.session.commit()
+
+    @pytest.fixture()
+    def create_working_alpha_report_schedule(self):
+        with self.create_app().app_context():
+
             alpha_user = self.get_user("alpha")
```

Comment:
there is a fixture called `login_as` might help for this case.

Reply:
Same here